api/Dockerfile

```diff
@@ -17,6 +19,14 @@ ARG CADDY_VERSION=2.6.2
 # "php" stage
 FROM php:${PHP_VERSION}-fpm-alpine${ALPINE_VERSION} AS api_platform_php
 
+# build for production
+ARG APP_ENV=prod
+
+WORKDIR /srv/api
+
+# php extensions installer: https://github.com/mlocati/docker-php-extension-installer
+COPY --from=mlocati/php-extension-installer --link /usr/bin/install-php-extensions /usr/local/bin/
```

Comment:
For the --link feature to work, you need to enable buildkit on your machine.
https://docs.docker.com/build/buildkit/

to add this to our Dockerfile, we also need to add a link to the installation notes to the documentation.
Specially that you need to use cli build with docker compose that buildkit is used:
https://stackoverflow.com/questions/58592259/how-do-you-enable-buildkit-with-docker-compose

The rebuild is a lot faster, cool

Reply:
I this specific to Linux? The documentation says that BuildKit is enabled by default for Docker Desktop, hence it worked for me out of the box.

Reply:
This is specific for the docker-ce (Community Edition).
With Docker Desktop it works out of the box. Github actions seems to have enabled buildkit already in the docker daemon.

Reply:
Building worked for me out of the box on Linux..? If this doesn't work everywhere, do we need to change something in the setup documentation? Or is it just a matter of a new enough Docker CE version?

Reply:
My understanding is that `docker compose build` uses docker/compose@v2 [which always uses buildkit by default](https://www.docker.com/blog/announcing-compose-v2-general-availability/).
Whereas `docker-compose build` uses docker/compose@v1, in which case either buildkit need to be enabled (automatically by Docker Desktop or manually via env variable) or by aliasing `docker-compose` to `docker compose` (which can be enabled in Docker Desktop settings).

So my naive understanding would be that this should always work out of the box when using `docker compose`. But cannot test on my side, as I'm using Docker Desktop.

Reply:
Hm but I am pretty sure that I am using it as docker-compose, since I have an alias `dcb` for `docker-compose build` on my system...
@BacLuc can you reproduce the problem consistently?